frontend/src/app/main/ui/shapes/text.cljs

```diff
@@ -22,10 +24,15 @@
 (mf/defc text-shape
   {::mf/wrap-props false}
   [props]
-  (let [{:keys [position-data content] :as shape} (obj/get props "shape")]
+  (let [{:keys [position-data content] :as shape} (obj/get props "shape")
+        is-component? (mf/use-ctx ctx/is-component?)]
 
     (mf/with-memo [content]
       (load-fonts! content))
 
-    (when (some? position-data)
-      [:> svg/text-shape props])))
+    (if is-component?
```

Comment:
The way this is put a component with position-data will be rendered the old way.

Reply:
Thanks! :)